MigFiles/SupportLibraries/ZWaveLib/ZWavePort.cs

```diff
@@ -123,6 +123,7 @@ public byte SendMessage(ZWaveMessage message, bool disableCallback = false)
                 if (disableCallback)
                 {
                     message.Message[message.Message.Length - 1] = GenerateChecksum(message.Message);
+                    Utility.logMessage("    Sent1   : " + Utility.ByteArrayToString(message.Message));
```

Comment:
@snagytx please remove these Utility.logMessage (the function as well), because there's already debug output from serial port.